api/MerchantValidationEvent.json

```diff
@@ -0,0 +1,271 @@
+{
+  "api": {
+    "MerchantValidationEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/MerchantValidationEvent",
+        "support": {
+          "chrome": {
+            "version_added": false
+          },
+          "chrome_android": {
+            "version_added": false
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": "64",
+            "notes": "Available only in Nightly builds."
+          },
+          "firefox_android": {
+            "version_added": "64",
+            "notes": "Available only in Nightly builds."
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": false
+          },
+          "opera_android": {
+            "version_added": false
+          },
+          "safari": {
+            "version_added": null
```

Comment:
The interface is exposed in SafariTP 12.1.

Reply:
Applies to all. Safari TP 12.1 fully conforms to the specification.

Reply:
No problem. I imagine they will ship to stable soon.

Reply:
@marcoscaceres Is it all new in TP 12.1? Or is it just that 12.1 fully conforms?

Reply:
It’s that 12.1 fully conforms.

Reply:
This will need to wait until 12.1 ships, since we don't currently have a way to deal with Safari Tech Preview builds. A follow-up PR will need to do it.